yesod-test/Yesod/Test.hs

```diff
@@ -24,8 +24,86 @@ This is very useful for testing web applications built in yesod, for example,
 where your forms may have field names generated by the framework or a randomly
 generated CSRF token input.
 
-Your database is also directly available so you can use 'runDB' to set up
+=== Example project
+
+The best way to see an example project using yesod-test is to create a scaffolded Yesod project:
+
+@stack new projectname yesod-sqlite@
+
+(See https://github.com/commercialhaskell/stack-templates/wiki#yesod for the full list of Yesod templates)
+
+The scaffolded project makes your database directly available in tests, so you can use 'runDB' to set up
 backend pre-conditions, or to assert that your session is having the desired effect.
+It also handles wiping your database between each test.
+
+=== Example code
```

Comment:
People are always asking for more high-level documentation in Haddocks, so I thought adding some sample code would be good. The scaffolded project is a more complete example, but if I'm just exploring yesod-test for the first time it's nice to have at-a-glance, hyperlinked haddocks.

Reply:
Personally, I think Haddock is a horrible format, and prefer putting this kind of documentation in a README.md. But no objection if you want to put up with this format.